src/scss/patterns/_sections.scss

```diff
@@ -69,7 +69,8 @@
 // ------
 
 [data-block] {
-  margin-block: var(--double-gutter);
+  margin-bottom: var(--double-gutter);
+  margin-top: var(--double-gutter);
```

Comment:
`var(--block-spacer, var(--double-gutter))`? Then below we could set `--block-spacer: 0` on `[data-grid]`?